### PR TITLE
Enha: Pin/Bundle sentry-cli Version

### LINKED ISSUE #136: Pin fastlane plugin version with CLI version
After discussion, it was decided that each Fastlane plugin version should use a specific CLI version to avoid new CLI releases breaking the plugin.

### PR DESCRIPTION
**Pull request by denrase**
Closes #136

# What changed?

- Pin version in `script/sentry-cli.properties`
  - We will also use this file to automatically create PRs when a new version is released.
- Introduce `script/sentry-cli-download.sh` to download the configures macOS & Win64 binaries.
- Bundle binary files with the gem by adding executable wrapper that returns the bundled binary path according to the OS.

# Whats next?

- [ ] Are there any drawbacks/regressions to this approach or something else we did not think of?
- [x] Version is pinned to `1.72.0` for now, need to create a separate PR to update to the most recent version.
- [x] Do we also need to support windows/linux? If so, we also need to bundle those binaries.
- [ ] Since we bundle `sentry-cli` now, we probably don't need `SentryCheckCliInstalledAction` anymore. Should we deprecate & remove it?

### CODE REVIEW

**Comment by denrase** [comment]
Relates to https://github.com/getsentry/team-mobile/issues/7

**Review by philipphofmann — COMMENTED** [review]
@denrase, please add a note to the ReadMe, that since version 1.13.0 we bundle sentry-cli but you can still use your own local installation. We should also point out how to use your local installation.

Apart from that LGTM, thanks 🙏

**Comment by denrase** [comment]
@philipphofmann We now also bundle the `Winn64` binary and i've extended the integration test GH action to also run on `windows-latest`.

File: .github/workflows/integration-test.yml [hunk 1]
```diff
@@ -18,11 +21,16 @@ jobs:
         with:
           ruby-version: '3.1.2'
 
-      - name: Install sentry-cli
-        run: curl -sL https://sentry.io/get-cli/ | bash
-
-      - name: Install Fastlane
+      - if: matrix.os == 'macos-latest'
+        name: Install Fastlane (macOS)
```
Comment (philipphofmann) [inline .github/workflows/integration-test.yml:25]: `l`: You can also use the matrix in the name, so you don't have to duplicate this.
```suggestion
      - if: matrix.os == 'macos-latest'
        name: Install Fastlane {{ matrix.os }}
```

**Review by philipphofmann — APPROVED** [review]
LGTM, thanks @denrase.

**Pull request merged**